src/plugins/data_source_management/public/components/data_source_view/data_source_view.tsx

```diff
@@ -137,10 +154,51 @@ export class DataSourceView extends React.Component<DataSourceViewProps, DataSou
   closePopover() {
     this.setState({ ...this.state, isPopoverOpen: false });
   }
+  getFilteredDefaultDataSource(defaultDataSource: string | null): string | null {
```

Comment:
nit: Need extra line here between closePopover and getFilteredDefaultDataSource.